narwhals/_pandas_like/expr.py

```diff
@@ -274,6 +278,14 @@ def ends_with(self, suffix: str) -> PandasExpr:
             suffix,
         )
 
+    def starts_with(self, prefix: str) -> PandasExpr:
+        return register_namespace_expression_call(
+            self._expr,
+            "str",
+            "starts_with",
+            prefix,
+        )
```

Comment:
this looks self-contained, and already correct - fancy splitting it out into a separate PR so we can merge it quickly?

Reply:
Very self contained :) Would you mind waiting for the rest? (Lazy mode activated from my side 😂)